setup.py

```diff
@@ -4,22 +4,11 @@
 
 from setuptools import setup, find_packages
 
-if __name__ == '__main__':
+if __name__ in ["__main__", "builtins"]:
```

Comment:
I've seen this conditional in some other aiida-related packages and generally consider this bad practice. Why do we even need this?

Reply:
Otherwise `pip-compile` fails for me.

Reply:
My question is: why do we need "`if __name__ ... ` ?

Reply:
I never thought about this, honestly. Just copied what AiiDA had at the time (and still has btw).

Reply:
Unless there is a good reason I suggest we remove this.

Reply:
I quickly had a look here: https://stackoverflow.com/questions/59831397/run-setup-function-from-setuptools-only-if-name-main

Makes sense to me to remove it, as we are not putting to-be-imported functions in this file

Reply:
done